### PR TITLE
Create 2024-09-20-quickstart-fde-yast2.md

### PR DESCRIPTION
**Pull request by thkukuk**
Quickstart Tutorial to use Tumbleweed with FDE and TPM2/Fido2

### CODE REVIEW

File: _posts/2024-09-20-quickstart-fde-yast2.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+author: Thorsten Kukuk
+date: 2024-09-20 09:00:00+02:00
+layout: post
+license: CC-BY-SA-3.0
+title:
```
Comment (tacerus) [inline _posts/2024-09-20-quickstart-fde-yast2.md:6]: Hi, is the empty title on purpose?

**Comment by tacerus** [comment]
Please move the file to one of the year subdirectories.

File: _posts/2024-09-20-quickstart-fde-yast2.md (hunk 1 above)
Comment (thkukuk) [reply _posts/2024-09-20-quickstart-fde-yast2.md]: No, no idea how the title got lost...

**Comment by thkukuk** [comment]
Next time I will use the git cli tools again instead the workflow...

**Review by tacerus — APPROVED** [review]

**Comment by lkocman** [comment]
I do have only one complain and that's the header image which seem to have a black line on the left. Let me try to correct it.
image

**Review by lkocman — APPROVED** [review]
Bg issue fixed in #271

**Pull request merged**